server/index.js

```diff
@@ -1,11 +1,16 @@
 require('dotenv').config();
 const express = require('express');
+const db = require('./psql/models.js');
 const app = express();
 app.use(express.json());
 
 app.get('/products', (req, res) => {
-  // TODO
-  res.send();
+  let { page, count } = req.query;
+  page = page || 1;
```

Comment:
@tchitrakorn I am sorry I missed it earlier. But I just found a bug in my code which is pretty similar to this. In Postman, if i made a request with page=0, then there was an error. If you face the same bug, then you can wrap the page and count with Number.

Reply:
@farhanali3193 Thank you for letting me know! I'll double check and look more into this.